plugins/applications/src/lib.rs

```diff
@@ -105,13 +105,33 @@ pub fn get_matches(input: RString, state: &State) -> RVec<Match> {
         .entries
         .iter()
         .filter_map(|(entry, id)| {
-            let score: i64 = matcher.fuzzy_match(&entry.name, &input).unwrap_or(0)
-                + matcher.fuzzy_match(&entry.exec, &input).unwrap_or(0)
-                + entry
-                    .keywords
-                    .iter()
-                    .map(|keyword| matcher.fuzzy_match(keyword, &input).unwrap_or(0))
-                    .sum::<i64>();
+            // For desktop actions we still want the match
+            // on the applications name, so we match the description.
+            // We substract the offset so that action entries are prioritized
+            // by order of definition if they match for the same application.
+            let name_score = match &entry.desc {
```

Comment:
I think we should include matching for both the name and the action, with the current version you can type in `libre new` which gives you the action for opening a new window for LibreWolf. With this PR that is broken.

Reply:
That was a logical error in general. Fixed it.

Reply:
I would probably make it so that the application name is before the desktop action name.

Reply:
Not sure what you mean. It is. Keep in mind that with desktop actions the name is the action name and the description is the applications name.

Reply:
Oh wait I'm just confusing myself, yeah it seems fine now.